diff --git a/src/components/nav-section/mini/nav-list.jsx b/src/components/nav-section/mini/nav-list.jsx
index 18dc7cb2316f1ee68a9d12dcb45743cde8980566..2505a561d245c41861d466916a68db293ed77fec 100644
--- a/src/components/nav-section/mini/nav-list.jsx
+++ b/src/components/nav-section/mini/nav-list.jsx
@@ -40,7 +40,7 @@ export default function NavList({ data, depth, slotProps }) {
         open={openMenu}
         onMouseEnter={handleOpenMenu}
         onMouseLeave={handleCloseMenu}
-        title={data.title}
+        title={data.compactTitle ?? data.title}
         path={data.path}
         icon={data.icon}
         info={data.info}
diff --git a/src/config/schemas.js b/src/config/schemas.js
index 964c20750b608af601ba0921a6d58f7d3f8ea711..6b6e1300ca5314dad116d167230d000c5462c510 100644
--- a/src/config/schemas.js
+++ b/src/config/schemas.js
@@ -55,6 +55,7 @@ export const MenuConfigSchema = Joi.object({
           .items(
             Joi.object({
               title: Joi.string().required(),
+              compactTitle: Joi.string().optional(),
               icon: Joi.string().required(),
               path: Joi.string().required(),
               external: Joi.boolean().optional().default(false),
